vars/notifyBuildResult.groovy

```diff
@@ -108,12 +119,20 @@ def call(Map args = [:]) {
           data['credentialId'] = slackCredentials
           data['enabled'] = slackNotify
           log(level: 'DEBUG', text: "notifyBuildResult: Notifying results in slack.")
-          notificationManager.notifySlack(data)
+          catchError(message: "There were some failures when notifying results slack", buildResult: 'SUCCESS', stageResult: 'UNSTABLE') {
+            notificationManager.notifySlack(data)
+          }
         }
         log(level: 'DEBUG', text: 'notifyBuildResult: Generate build report.')
         catchError(message: "There were some failures when generating the build report", buildResult: 'SUCCESS', stageResult: 'UNSTABLE') {
           notificationManager.generateBuildReport(data)
         }
+
+        if (aggregateComments) {
```

Comment:
What will happen if `aggregateComments` is false? Should it loop the `notifications` array?

Reply:
That's delegated to each specific method, since there is a flag to enable/disable the notifications:
- https://github.com/elastic/apm-pipeline-library/pull/841/files#diff-e79413d8e910494d1fa40002bd01bb66c3830ab0646a68e93e15c4d43a7a22e7R77

In other words, the `notifyBuildResult` step is in charge to pass that particular configuration to each notifier to create the GH comment, or if it's aggregated then it creates the aggregated comment

Reply:
Ah ok ok, because it's inside the data, it will be managed on each specific method